docs/3____physical_signal_abstraction.adoc

```diff
@@ -124,6 +124,9 @@ All <<high-cut-terminal-member-variable-for-signals,`TerminalMemberVariables`>>
 <<high-cut-pdu-terminal,PDU terminals>> list all the contained signals as `<TerminalMemberVariable>`.
 
 Attribute definitions::
+ * `variableName` refers to the input or output variable name of the FMU.
+    These variables represent the signal abstraction layer ("high cut").
+    Unless there are other requirements, it is recommended to build the variable names as follows: `BusName.FrameName.PDUName.SignalName`, e.g., `Powertrain.tcuSensors.tcuSensors.vCar`.
  * `variableName` refers to the input or output variable name of the FMU and to enforce uniqueness is built as follows: `BusName.FrameName.PDUName.SignalName`, e.g., `Powertrain.tcuSensors.tcuSensors.vCar`.
```

Comment:
I think the adjustments are good. But shouldn't this block be removed?

Reply:
> I think the adjustments are good. But shouldn't this block be removed?

Yes, this was an edit oopsie, the first paragraph should have been a replacement, as you guessed.
 
> Maybe we should move the "uniqueness" req. to the first (new) list item

I have added a further sentence to provide more of this rationale information, and also mention clock-variables, as we now (with the other PR) will also not require speicific variable names there. Maybe we could also move this recommentation to a higher level?

Reply:
> > I think the adjustments are good. But shouldn't this block be removed?
> 
> Yes, this was an edit oopsie, the first paragraph should have been a replacement, as you guessed.
> 
> > Maybe we should move the "uniqueness" req. to the first (new) list item
> 
> I have added a further sentence to provide more of this rationale information, and also mention clock-variables, as we now (with the other PR) will also not require speicific variable names there. Maybe we could also move this recommentation to a higher level?

Yes I think so, we can also move it on the top, because its relevant for every variable/clock within the high-cut. I'm just not sure if we really need to mention it, because 

> > I think the adjustments are good. But shouldn't this block be removed?
> 
> Yes, this was an edit oopsie, the first paragraph should have been a replacement, as you guessed.
> 
> > Maybe we should move the "uniqueness" req. to the first (new) list item
> 
> I have added a further sentence to provide more of this rationale information, and also mention clock-variables, as we now (with the other PR) will also not require speicific variable names there. Maybe we could also move this recommentation to a higher level?

Yes for sure we can move it. The only thing is that I'm quite not sure if this is already clear from the FMI 3 standard side and we an just leave it out. What do you think?